Project.toml

```diff
@@ -1,12 +1,13 @@
+
```

Comment:
Hmm, that's new. Something must be off in the `sed` command. Can you try to modify it so that we don't get this spurious newline.

Reply:
Ha, nevermind.

Looks like the current `main` version 

https://github.com/etpinard/dash-textarea-autocomplete/blob/main/Project.toml#L1

has a newline at the top of file also.

Reply:
yeah that would be part of a PR to https://github.com/plotly/dash/blob/dev/dash/development/_jl_components_generation.py 

looking at it, it seems like https://github.com/plotly/dash/blob/dev/dash/development/_jl_components_generation.py#L87 is part of `main` , so why aren't we getting that with `npm run build` ? 🤔

Reply:
> looking at it, it seems like https://github.com/plotly/dash/blob/dev/dash/development/_jl_components_generation.py#L87 is part of main , so why aren't we getting that with npm run build ? thinking

That's their `dev` branch, so yeah it will be part of the next `dash.py` release.

Reply:
oooh -- right, sorry.